plugins/ci/pkg/ci/ci.go

```diff
@@ -50,15 +50,21 @@ func GetResultsFromCommand(command string, args ...string) (string, error) {
 	return strings.TrimSpace(string(bytes)), err
 }
 
-// GetImagesFromManifest scans a folder of yaml files and returns all of the images used.
-func GetImagesFromManifest(configFolder string) ([]trivymodels.Image, []models.Resource, error) {
+// GetAllResources scans a folder of yaml files and returns all of the images and resources used.
+func GetAllResources(configFolder string, configurationObject models.Configuration) ([]trivymodels.Image, []models.Resource, error) {
 	images := make([]trivymodels.Image, 0)
 	resources := make([]models.Resource, 0)
 	err := filepath.Walk(configFolder, func(path string, info os.FileInfo, err error) error {
 		if !strings.HasSuffix(info.Name(), ".yaml") {
 			return nil
 		}
-		relativePath, err := filepath.Rel(configFolder, path)
+		filefolder := configFolder
+		for _, helmObject := range configurationObject.Manifests.Helm {
+			if strings.HasPrefix(filefolder, helmObject.Name) {
```

Comment:
I think this would be a bit safer if we did `strings.HasPrefix(filefolder, helmObject.Name + "/")`

Reply:
If we hit this block, can we save `helmObject.Name` as `resource.HelmName` (at lines 106 and 124 below)